installer/win/Brackets_en-us.wxl

```diff
@@ -55,4 +55,6 @@
     <String Id="VerifyReadyDlgInstallText" Overridable="yes">Click Install to begin, or Cancel to exit.</String>
     <String Id="VerifyReadyDlgRemoveText" Overridable="yes">Click Remove to uninstall !(loc.ShortProductName), or Cancel to exit.</String>
     <String Id="OpenWithText" Overridable="yes">Open With !(loc.ShortProductName)</String>
+    <String Id="AddBracketsToPath" Overridable="yes">Update PATH environment variable with !(loc.ShortProductName) install location (in order to launch !(loc.ShortProductName) from command line)</String>
+    <String Id="AddContextMenu" Overridable="yes">Add &quot;Open With !(loc.ShortProductName) &quot; menu to file and folder context menus</String>
```

Comment:
@nethip How about if we take out the word "shortcut" (since you're right -- installers also often refer to Start menu shortcuts, etc., so it could be confusing)?

> Add "brackets" launcher to PATH for command line use

This is a good deal more concise than the existing string, and it's more correct since it show the command in the canonical lower-case...